core-java/src/test/java/org/baeldung/java/sorting/SetSort.java

```diff
@@ -0,0 +1,54 @@
+package org.baeldung.java.sorting;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.TreeSet;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class SetSort {
+
+    private HashSet<Integer> integers;
+    private TreeSet<Employee> employees;
```

Comment:
Since we have to show how to sort a Set, I thought reader would find it interesting to see both example. So I used HashSet in one example and TreeSet in Second so that we could cover both kinds of it.